### PR TITLE
Support the revision flag for `state install`

### PR DESCRIPTION
**Pull request by Naatan**
<table>
<td>
  <a href="https://activestatef.atlassian.net/browse/DX-2589" title="DX-2589" target="_blank">StoryDX-2589</a>  As a user I can specify a revision when installing packages
  </td></table>
  <br />

### CODE REVIEW

File: internal/locale/locales/en-us.yaml
Comment (MDrakos) [inline internal/locale/locales/en-us.yaml:1500]:
```suggestion
    You can install this package by running `[ACTIONABLE]state install {{.V1}}/{{.V0}} --r {{.V2}} --ts now`[/RESET].
```

It should be V2 right?

**Review by MDrakos — CHANGES_REQUESTED** [review]
This all looks good to me. Just requesting changes as the integration tests haven't run and we should do that before merging.

File: internal/locale/locales/en-us.yaml
Comment (Naatan) [reply internal/locale/locales/en-us.yaml:1500]: No, V2 holds the version, which is used above this line (it's a multiline locale).

**Comment by Naatan** [comment]
> This all looks good to me. Just requesting changes as the integration tests haven't run and we should do that before merging.

The run is over here: https://github.com/ActiveState/cli/actions/runs/7980174137?pr=3119

Github seems bugged in its reporting.. :\

**Comment by MDrakos** [comment]
> > This all looks good to me. Just requesting changes as the integration tests haven't run and we should do that before merging.
> 
> The run is over here: https://github.com/ActiveState/cli/actions/runs/7980174137?pr=3119
> 
> Github seems bugged in its reporting.. :\

That Link is just the `Target & Verify` job. But I will leave it up to you.

**Review by MDrakos — APPROVED** [review]

**Comment by Naatan** [comment]
> > > This all looks good to me. Just requesting changes as the integration tests haven't run and we should do that before merging.
> > 
> > 
> > The run is over here: [ActiveState/cli/actions/runs/7980174137?pr=3119](https://github.com/ActiveState/cli/actions/runs/7980174137?pr=3119)
> > Github seems bugged in its reporting.. :\
> 
> That Link is just the `Target & Verify` job. But I will leave it up to you.

Woops, copied the wrong one. Fwiw: https://github.com/ActiveState/cli/actions/runs/7980174129?pr=3119

**Pull request merged**
